content/tutorials/spatial_crs_coding/index.Rmd

```diff
@@ -31,46 +30,64 @@ options(stringsAsFactors = FALSE)
 
-### How _was_ a CRS represented in R? Evolutions in PROJ and GDAL.
+### How _did_ we represent a CRS in R? Evolutions in PROJ and GDAL.
 
 It is good to know this, but you _can_ skip this section if you like.
 
 The reason for writing this tutorial are the recent (and ongoing) changes in several important geospatial libraries, especially **GDAL** and **PROJ**.
-They are used by most geospatial tools, including the key geospatial R packages `rgdal`, `sp`, `sf` and `raster`.
+They are used by most geospatial tools, including the key geospatial R packages `rgdal`, `sp`, `sf`, `stars`, `terra` and `raster`.
 
 Since long, coordinate reference systems in R (and many other tools) have been represented by so called 'PROJ.4 strings' (or 'proj4strings'), referring to the long-standing version 4 of the PROJ library.
-But, we will not use them here - it is counteradvised to use them any longer!
+But, we will not use them here!
+It is discouraged to use 'PROJ strings'[^projstring] any longer to represent a CRS; several string elements for CRSs are now deprecated or unsupported.
+Currently, PROJ (<https://proj.org>) regards PROJ strings only as a means of specifying a _coordinate operation_ (conversions or transformations _between_ CRSs).
+Performing coordinate operations is the main aim of PROJ.
 
-Let's just have **one last nostalgic peek** (and then, no more!!): the proj4string for the Belgian Lambert 72 CRS (EPSG:31370) "_was_":
+[^projstring]: 'PROJ string' is the term used in current PROJ documentation.
+Here, we only use 'PROJ.4 string' (or 'proj4string') when referring specifically to the PROJ string appearance in version 4 of the PROJ software.
+
+Let's just have **one last nostalgic peek** (and then, no more!!) to the proj4string for EPSG:31370, the Belgian Lambert 72 CRS [^belge72]:
 
     +proj=lcc +lat_1=51.16666723333333 +lat_2=49.8333339 +lat_0=90 +lon_0=4.367486666666666 +x_0=150000.013 +y_0=5400088.438 +ellps=intl +towgs84=-106.8686,52.2978,-103.7239,0.3366,-0.457,1.8422,-1.2747 +units=m +no_defs
 
-Several reasons, such as WGS84 getting outdated, higher accuracy requirements in transformations and the availability of better standards than proj4strings, have led to recent changes in PROJ and GDAL.
-This also implied reduced compatibility with proj4strings.
+[^belge72]: Note that the _currently returned_ PROJ string for EPSG:31370, if requested from PROJ ≥ 6 or GDAL 3 (not shown), lacks the datum reference which, in PROJ.4, was defined indirectly by `+towgs84` with the 7-parameter (Helmert) transformation to the WGS84 datum.
+Hence the current PROJ string is a deficient representation of EPSG:31370.
+
+Several reasons have led to recent changes in GDAL and PROJ, such as the use of WGS84 as a transformation hub while it is a dynamic CRS, higher accuracy requirements in transformations and the availability of better CRS specification standards than PROJ strings.
```

Comment:
Several difficult terms here: 'transformation hub' and 'dynamic CRS' will not be clear for most readers.

Reply:
Do these terms relate to the 'ensemble datum' which you explained in a footnote?

Reply:
I'll try to make that a bit more clear (or add a reference), but I feel a bit obliged here to not explain in full - it would need much more text while it is just background information.

Reply:
8a336571